### PR TITLE
Feature feedback suggestions settings

### PR DESCRIPTION
**Pull request by faloi**
Related to mumuki/mumuki-platform#283.

image

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -40,20 +40,21 @@
...
-    t.datetime "created_at",                               null: false
-    t.datetime "updated_at",                               null: false
-    t.text     "books",                    default: [],                 array: true
+    t.datetime "created_at",                                   null: false
+    t.datetime "updated_at",                                   null: false
+    t.text     "books",                        default: [],                 array: true
     t.string   "locale"
     t.text     "terms_of_service"
-    t.text     "login_methods",            default: [],                 array: true
+    t.text     "login_methods",                default: [],                 array: true
```
Comment (flbulgarelli) [inline db/schema.rb]: Please update the DB schema, so that it doesn't duplicate the structure of Mumukit::Organization. See laboratory latest schema.

Comment (flbulgarelli) [inline db/schema.rb]: Now the organizations table should have a  `settings` text field, with the `Mumukit::Platform::Organization::Settings` json embedded.

Comment (faloi) [inline db/schema.rb]: Great! I was a bit disappointed when I noticed the duplication, but my faith in you is now restored. :smile:

Comment (faloi) [inline db/schema.rb]: Well, this is not a minor change, so I propose to do it in another pull request. Do you agree?

Comment (flbulgarelli) [inline db/schema.rb]: I'd rather not. Doing this change this way is a source of lots of eventual integration problems. 

Nevertheless, we are not going to merge this PR right now since we want to first tag `mumukit-platform` and pospone any change to office model until version `4.5` - that is, in two versions, in order to allow the other apps to migrate to the new schema first - so there is no hurry :smile:

Comment (faloi) [inline db/schema.rb]: Ok. Self-reminder: see https://github.com/mumuki/mumuki-laboratory/pull/798

**Comment by fedescarpa** [comment]
Esto está desactualizado, deberíamos reveerlo mas tarde
